smarts/core/agent_interface.py

```diff
@@ -151,6 +151,12 @@ class DoneCriteria:
     """
 
 
+@dataclass
+class AgentBehavior:
+
```

Comment:
Should have a docstring since it will be in one of the outward facing interfaces.